plugins/generic/htmlArticleGalley/display.tpl

```diff
@@ -7,7 +7,36 @@
  *
  * Embedded viewing of a HTML galley.
  *}
-{foreach from=$styleUrls item=styleUrl}
-	<link href="{$styleUrl|escape}" media="all" type="text/css" rel="stylesheet"/>
-{/foreach}
-{$htmlGalleyContents}
+<!DOCTYPE html>
+<html lang="{$currentLocale|replace:"_":"-"}" xml:lang="{$currentLocale|replace:"_":"-"}">
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset={$defaultCharset|escape}" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>{translate key="article.pageTitle" title=$title}</title>
+
+	{load_header context="frontend" headers=$headers}
+	{load_stylesheet context="frontend" stylesheets=$stylesheets}
+	{load_script context="frontend" scripts=$scripts}
+</head>
```

Comment:
Super minor thing, but could you use `{include file="core:common/headerHead.tpl"}` here?